src/kafka-tools/Dockerfile

```
@@ -6,6 +6,6 @@ RUN wget https://archive.apache.org/dist/kafka/3.8.0/kafka_2.13-3.8.0.tgz
 RUN tar -xzf kafka_2.13-3.8.0.tgz
 RUN ln /opt/kafka_2.13-3.8.0/bin/kafka-topics.sh /bin/kafka-topics
 WORKDIR /opt/kafka_2.13-3.8.0/
-COPY ssl.config kafka.sh list.txt .
+COPY ssl.config kafka.sh fix_replication.sh list.txt .
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)